src/devices/Max7219/Font.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Max7219
+{
+    public class Font
+    {
+        readonly byte[][] _bitmap;
+
+        public Font(params byte[][] bitmap)
+        {
+            _bitmap = bitmap;
+        }
+
+        public byte[] GetItem(char chr)
+        {
+            var bitmap = _bitmap[chr];
```

Comment:
>var bitmap = _bitmap[chr]; [](start = 12, length = 26)

is it ok to get out of range exception here? I feel the design of this class is not good enough.

Reply:
I've added a few comments and think it's clear now.